students/shibin_mathew/lesson9/mailroom.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+import sys
+from collections import defaultdict
+
+
+class Donor:
+    def __init__(self, names, amount):
+        self.names = names
+        self.amount = amount
+
+    @property
+    def names(self):
+        return "{}".format(self._names)
+
+    @names.setter
+    def names(self, names):
+        self._names = names
+
+    @property
+    def amount(self):
+        return self._amount
+
+    @amount.setter
+    def amount(self, amount):
+        self._amount = amount
+
+
+class Collection:
+    def __init__(self, donors2=None):
+        self.donors2 = donors2
+
+    @property
+    def donors2(self):
+        return self._donors2
+
+    @donors2.setter
+    def donors2(self, donors2):
+        self._donors2 = donors2
+
+    def print_donor_names(self, donors2):
+        list_test = []
+        for names in donors2:
+            list_test.append(names)
+        return list_test
+
+    def send_thank_you(self, donors2):
+        #  Function to Send a Thank You Email
+        c1 = Collection()
+        program_quit = False
+        while not program_quit:
+            name = input("Please input your full name\n")
```

Comment:
remember the separation of concern from the assignment page, you should not have any `input` functions in your data handling code

Reply:
I created a class to handle the question module which will separate the functions that take in the inputs with the functions that handle the data.